DEVELOPER.md

```diff
@@ -285,6 +295,26 @@ This will retain any files created by the test in `exports_scratch` in the curre
 
 There is also a `--keep-failed <dir>` option which preserves the outputs of failed tests in the given location, however this option is only compatible with function-scoped temporary directories (the `function_tmpdir` fixture).
 
+#### Debugging tests in VS Code
+When writing tests to develop a new feature or reproduce and fix a bug, it can often be helpful to debug tests interactively in an IDE. In addition to the [documentation](https://code.visualstudio.com/docs/python/testing), the following tips might be helpful for getting test debugging to work in VS Code:
+
+* Add the following to the `settings.json` file:
+
+    `"python.testing.pytestArgs": ["."],`  
+    `"python.testing.unittestEnabled": false,`  
+    `"python.testing.pytestEnabled": true,`  
+    `"python.testing.cwd": "${workspaceFolder}/autotest"`  
+    The first three may be already set correctly by default, but the last item is needed for VS Code to discover the tests correctly and run the tests from the `autotest` folder.
```

Comment:
Maybe worth mentioning these can also be set with cmd palette `Python: Configure Tests`